build.properties

```diff
@@ -10,10 +9,12 @@ mod_url=https://github.com/Team-RTG/Realistic-Terrain-Generation
 mod_author="Team RTG"
 mod_creds=Forked from RWG by ted80. OpenSimplex Noise by KdotJPG.
 mod_logo=assets/rtg/logo.png
-mcf_version=12.18.1.2011
-# mcf_postfix is the branch postfix (without '-') on the Forge version when it's not the default branch in the Forge repo
-# This should be same as mc_version, eg Non-default: 1.9.4-12.17.0.1974-1.9.4, Default: 1.9.4-12.17.0.1976
-mcf_postfix=
-mappings=snapshot_nodoc_20160724
+mcf_version=12.18.1.2094
```

Comment:
> This seems irrelevant, since Block#getStateFromMeta is deprecated and just returns Block#getDefaultState regardless of the meta value passed.

I was under the impression that you are supposed to override Block#getStateFromMeta(), and not rely on that default implementation for anything -- i.e, it is only there to make sure that something non-crashing happens.